src/util.py

```diff
@@ -17,13 +17,15 @@ def resource_path(relative):
 
 def get_resource_path():
     """Get the path to the resources folder."""
-    resource_folder = os.path.join(os.path.dirname(os.path.dirname(__file__)), 'resources')
+    resource_folder = os.path.join(
```

Comment:
I realise this wasn't you who added this but would be a good idea to switch to the Path library rather than os.path. That would probably be a good issue to open.

Reply:
Haven't worked with it before so not confident enough on putting it in the mr, a separate issue does seem more appropriate